src/components/Comments/ReplyCommentItem.js

```diff
@@ -217,6 +290,17 @@ const ReplyCommentItem = ({
             <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
           </ButtonHightlight>
         </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => refSheet.current.open()}
+          testID="sendDMbtn"
+          activeOpacity={1}>
+          <ButtonHightlight
+            onLongPress={handleLongPress}
+            onPress={() => refSheet.current.open()}
+            style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
+            <MemoSendDM />
+          </ButtonHightlight>
+        </TouchableOpacity>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new button for opening the bottom sheet menu has been added. The `onPress` function is duplicated in the `TouchableOpacity` and `ButtonHightlight` components. This redundancy can be avoided by removing the `onPress` from one of them.



<!-- This is an auto-generated comment by OSS CodeRabbit -->